### PR TITLE
Features/daily record

### LINKED ISSUE #10: Send mutations for Daily Record at end of day, or create a submit button
- [ ] Send a mutation request to BE. Create a button to submit shower data, or send the end of day mutation request with "Today's Time"
- [ ] Include plastic usage in this post.

### LINKED ISSUE #11: Request other times from BE
Create queries to request a user's average and weekly total from the BE. Might need to check with BE to make sure this data is ready to go.

### PR DESCRIPTION
**Pull request by nbalmoori**
# Description

This branch creates a mutation query to allow a user to submit a daily record of their shower time and item usage.

## Issues

Closes #10
Closes #11 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

## How Has This Been Tested?

- [x] console.log()
- [x] dev tools
- [x] functionality check through web browser
- [ ] tested by viewing the markdown file preview
- [ ] tested with testing framework

## Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have deleted all comments/console.log() after functionality and understanding is final
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "dayjs": "^1.11.3",
```
Comment (whiteheadol) [inline package.json:14]: Wow, I can't believe this is the first time I've seen this!

File: src/components/ItemCounters.js [hunk 2]
```diff
@@ -12,6 +17,24 @@ const ItemCounters = ({ containerCount, setContainerCount, bagCount, setBagCount
     }
   }
 
+  const [createDailyRecord, { error }] = useMutation(SUBMIT_RECORD)
+
+  const submitRecord = () => {
```
Comment (whiteheadol) [inline src/components/ItemCounters.js:22]: Look at us go!

**Review by whiteheadol — APPROVED** [review]

File: src/components/ItemCounters.js [hunk 3]
```diff
@@ -12,6 +17,24 @@ const ItemCounters = ({ containerCount, setContainerCount, bagCount, setBagCount
...
+        userId: 8,
+        bagCount: bagCount,
+        containerCount: containerCount,
+        strawCount: strawCount,
+        showerTime: showerTime
+      }
+    })
+    if (error) {
+      console.log(error);
+    }
```
Comment (rachellhill) [inline src/components/ItemCounters.js:35]: Great work!!

File: src/components/Queries.js [hunk 4]
```diff
@@ -11,4 +11,28 @@ export const GET_USER = gql`
...
+        date: $date
+        userId: $userId
+        bagCount: $bagCount
+        containerCount: $containerCount
+        strawCount: $strawCount
+        showerTime: $showerTime
+      }) {
+        containerCount
+        date
+        showerTime
```
Comment (rachellhill) [inline src/components/Queries.js:35]: This looks great! Maybe we could move all mutations to a separate mutations file!

**Review by rachellhill — APPROVED** [review]
Great work!!

**Pull request merged**